cfgov/core/tests/test_management_inactive_users.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from StringIO import StringIO
+
+from datetime import timedelta
+
+import mock
+
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils import timezone
+from django.utils.formats import date_format
+
+from core.management.commands.inactive_users import Command
```

Comment:
I put the django imports before the local imports... I think I'd prefer to keep it that way, unless you feel strongly otherwise.